lib/mixlib/shellout/windows.rb

```diff
@@ -289,35 +288,17 @@ def should_run_under_cmd?(command)
         return false
       end
 
-      def pathext
-        @pathext ||= ENV["PATHEXT"] ? ENV["PATHEXT"].split(";") + [""] : [""]
-      end
-
-      # which() mimicks the Unix which command
-      # FIXME: it is not working
```

Comment:
yeah, its totes fun to try to figure out why.

i don't doubt it isn't working, but its a mess and hosh's tests are very stubby and fairly clever and hard to discern how all the pieces fit together.